app/assets/javascripts/revised/pages/bikes/edit_alert.coffee

```diff
@@ -2,11 +2,37 @@ class BikeIndex.BikesEditAlert extends BikeIndex
   constructor: ->
     super()
     @$planSelectionForm = $('#js-select-plan-form')
+    @$imageSelectionContainer = $('#js-select-image-container')
     @initializeEventListeners()
 
```

Comment:
Can you render with the first image by default on page load?

Reply:
Good call! updated in fd492e51